src/js/base/helpers.cy.js

```diff
@@ -0,0 +1,173 @@
+import Backbone from 'backbone';
+import { View } from 'marionette';
+
+import hbs from 'handlebars-inline-precompile';
+
+import { testTs } from 'helpers/test-timestamp';
+import formatDate from 'helpers/format-date';
+
+context('Handlebars helpers', function() {
+  const testDate = testTs();
+
+  const MatchTextView = View.extend({
+    template: hbs`
+      <div class="test-element">
+        {{matchText "Patient Name" null}}
+      </div>
+      <div class="test-element">
+        {{matchText "Patient Name" "Patient"}}
+      </div>
+    `,
+  });
+
+  const DateTimeView = View.extend({
+    template: hbs`
+      <div class="test-element">
+        {{formatDateTime null "lll"}}
+      </div>
+      <div class="test-element">
+        {{formatDateTime testDate "lll"}}
+      </div>
+      <div class="test-element">
+        {{formatDateTime testDate "lll" nowrap=false}}
+      </div>
+      <div class="test-element">
+        {{formatDateTime null "lll" defaultHtml="No Date Available"}}
+      </div>
+    `,
+    templateContext() {
+      return {
+        testDate,
+      };
+    },
+  });
+
+  const PhoneView = View.extend({
+    template: hbs`
+      <div class="test-element">
+        {{formatPhoneNumber null}}
+      </div>
+      <div class="test-element">
+        {{formatPhoneNumber phone}}
+      </div>
+      <div class="test-element">
+        {{formatPhoneNumber badPhone}}
+      </div>
+      <div class="test-element">
+        {{formatPhoneNumber null defaultHtml="No Phone Available"}}
+      </div>
+    `,
+    templateContext() {
+      return {
+        phone: '6155555551',
+        badPhone: 'UNKNOWN',
+      };
+    },
+  });
+
+
+  specify('Match text formatting', function() {
+    const matchTextView = new MatchTextView({
+      model: new Backbone.Model(),
+    });
+
+    cy
+      .mount(rootView => {
+        return matchTextView;
+      })
+      .as('root');
+
+    cy
+      .get('@root')
+      .find('.test-element')
```

Comment:
I think we might want to name these instead of filtering them so that in the future tests can be added without modifying other tests by having to change `last` or updated `eq(#)` below where I inserted one.  So explicitly this'd be `.find('.test-null')` or implicitly it's just next.. should... next... should...

Reply:
Agreed, updated 👍